utils/react/Hooks.js

```diff
@@ -32,77 +31,38 @@ export function useOnUnmount(fn) {
 }
 
 /**
- * Hook to run a function when component is resized.
- * This will not run the hook when the size is changed to 0 or is changed back from 0 to a previous
- * size. This is to improve performance by avoiding unneeded resizing.
+ * Hook to run a function when a dom element is resized.
+ *
  * @param {function} fn
  * @param {Object} [c] - configuration object
  * @param {number} [c.debounce] - milliseconds to debounce
- * @param {Ref} [c.ref] - existing ref to observe. If not provided, a ref will be created
  * @returns {Ref} - ref to be placed on target component
```

Comment:
Is this return doc still correct now that we are returning a callback ref and not an actual `Ref`?

Reply:
Also might want a `@see observeResize()` doc comment here